src/test/java/api/signup/UpdateUserPositiveTest.java

```diff
@@ -0,0 +1,150 @@
+package api.signup;
+
+import org.realworld.api.Session;
+import org.realworld.api.datamodel.requests.UpdateUser;
+import org.realworld.api.datamodel.requests.UpdateUserRequest;
+import org.realworld.api.datamodel.responses.ResponseWrapper;
+import org.realworld.api.datamodel.responses.UserResponse;
+import org.realworld.api.services.ApiService;
+import org.realworld.utils.ResponseHandler;
+import org.realworld.utils.RetrofitFactory;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import static org.realworld.api.Constants.EMAIL_SUFFIX;
+import static org.realworld.api.Constants.USER_NAME_PREFIX;
+
+public class UpdateUserPositiveTest {
+
+    private ApiService authorizedApiService;
+
+    @BeforeMethod(alwaysRun = true)
+    public void SignIn() {
```

Comment:
is it common to name methods with capital letter in TestNG? Because usually, methods in Java a starting from lower case

Reply:
done